.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+Documentation:
+   Enabled: false
+LineLength:
+   Max: 99
```

Comment:
EOF, not a big deal but still nice to fix. Check if your text editor can do this by default :)